src/config/config.cpp.in

```diff
@@ -21,4 +21,4 @@ const std::string nmodl::Version::NMODL_VERSION = "@PROJECT_VERSION@";
  * from CMAKE_INSTALL_PREFIX.
  */
 std::vector<std::string> nmodl::NrnUnitsLib::NRNUNITSLIB_PATH =
-    {"@CMAKE_INSTALL_PREFIX@/share/nrnunits.lib", "@NMODL_PROJECT_SOURCE_DIR@/share/nrnunits.lib"};
+    {"@CMAKE_INSTALL_PREFIX@/share/nrnunits.lib", "@NMODL_PROJECT_BINARY_DIR@/share/nrnunits.lib"};
```

Comment:
```suggestion
    {"@CMAKE_INSTALL_PREFIX@/share/nrnunits.lib", "@CMAKE_CURRENT_BINARY_DIR@/share/nrnunits.lib"};
"""
Maybe this needs to be fixed as well? This is probably the reason why the unit tests are failing and maybe this is the reason behind the other issue as well

Reply:
`NMODL_CURRENT_BINARY_DIR` is already on `CMAKE_CURRENT_BINARY_DIR`, right?

Reply:
Actually no, those are the same..